repl2.html

```diff
@@ -19,5 +19,77 @@
 - repl
 ---
 <link rel="stylesheet" href="https://unpkg.com/codemirror@5.28.0/lib/codemirror.css">
+
 <noscript>You need to enable JavaScript to run this app.</noscript>
-<div id="root" class="babel-repl-v2">Loading...</div>
+
+<div id="root" class="babel-repl-v2">
+  <div class="repl-loading">
+    Loading Babel ...
+    <div class="repl-loading-animation-container">
+      <div class="repl-loading-animation-box repl-loading-animation-box-1"></div>
+      <div class="repl-loading-animation-box repl-loading-animation-box-2"></div>
+      <div class="repl-loading-animation-box repl-loading-animation-box-3"></div>
+      <div class="repl-loading-animation-box repl-loading-animation-box-4"></div>
+      <div class="repl-loading-animation-box repl-loading-animation-box-5"></div>
+    </div>
+  </div>
+</div>
+
+<style type="text/css">
+  @keyframes repl-loading-animation {
```

Comment:
Put this in `_sass/pages/_repl.scss`? We've already loaded the CSS by this point, as it's in the `<head>`. May as well put this in there 😃